### DISCUSSION TITLE
Feature Request: sort shopping list

### DISCUSSION THREAD

**Discussion by jmawet** [TandoorRecipes/recipes#3199] [Ideas]
I love that the shopping list groups by category, but it would be nice if we could manually change the order of items within each category! Some sorting/reorder functionality would be nice

**Comment by vabene1111** [comment]
thanks for the feebdack, I always wanted that and its also supported in the database, the shopping UI is just so cluttered already that I did not want to add any more to it. I will look into this again on the new version

**Comment by Timurling** [comment]
I support this reqeust. Each store is organized a little different and it would be greate to have the list organoized in the way you walk through the store.

**Reply by vabene1111** [reply]
you can do that pretty good with categories already. If your stuff is sorted into categories even if I go shopping for a week I rearely have more than 8-10 entries per category which can be looked over quickly (altough @jmawet still is right that in category sorting would be cool)
